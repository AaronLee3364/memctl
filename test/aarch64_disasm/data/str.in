fffffff0070c0000
f9000fff 790023e9 f9000008 39003be9
b80b4e7f b8004429 f81f8c86
