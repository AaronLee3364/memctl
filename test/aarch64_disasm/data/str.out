fffffff0070c0000  STR     XZR, [SP, #24]
fffffff0070c0004  STRH    W9, [SP, #16]
fffffff0070c0008  STR     X8, [X0]
fffffff0070c000c  STRB    W9, [SP, #14]
fffffff0070c0010  STR     WZR, [X19, #180]!
fffffff0070c0014  STR     W9, [X1], #4
fffffff0070c0018  STR     X6, [X4, #-8]!
